src/wallet/test/wallet_test_fixture.cpp

```diff
@@ -0,0 +1,28 @@
+#include "wallet/test/wallet_test_fixture.h"
+
+#include "rpc/server.h"
+#include "wallet/db.h"
+#include "wallet/wallet.h"
+
+WalletTestingSetup::WalletTestingSetup(const std::string& chainName):
+    TestingSetup(chainName)
+{
+    bitdb.MakeMock();
+
+    bool fFirstRun;
+    pwalletMain = new CWallet("wallet.dat");
```

Comment:
Could make sense to use "wallet_test.dat" here, to indicate that the file name is not hardcoded?